src/components/CheckoutForm.js

```diff
@@ -0,0 +1,107 @@
+import React, { Component } from "react";
+import {
+  CardNumberElement,
+  CardExpiryElement,
+  CardCvcElement,
+  injectStripe
+} from "react-stripe-elements";
+import axios from "axios";
+import { Form, Button, Card, Container } from "semantic-ui-react";
+
+class CheckoutForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      success: "",
+      error: "",
+      renderStripeForm: true
+    };
+    this.submitPayment = this.submitPayment.bind(this);
+  }
+
+  async submitPayment() {
+    let token = await this.props.stripe.createToken();
+    let response = await axios.post("/subscriptions", {
+      body: token.id
+    });
+    
+    if (response.data.message) {
```

Comment:
I would like to see the two if statements in this method to be an if-else conditional.

Reply:
Fixed